goreleaser.yml

```diff
@@ -64,6 +64,8 @@ archives:
       - README.md
       - LICENSE
       - contrib/gitlab.tpl
+      - contrib/junit.tpl
+      - contrib/sarif.tpl
```

Comment:
Maybe it is a good idea to make this future proof for any other additional template file.
Analog to #619